src/node_api.js

```diff
@@ -1,6 +1,6 @@
-import {DEFAULT_QUIZ_CODE_LINK} from "./configurations";
+import {DEFAULT_QUIZ_LINK} from "./configurations.example";
```

Comment:
oops. it is my bad that this gets confusing.
you are supposed to import from `./configurations`.
But now I think of it, the build will fail, so you have two choices:
* Change the build configuration, so it fetches a configurations.js file.
* Rename configurations.example.js to configurations.js.

Whichever you think makes the most sense, the point of having an example is so we don't expose sensitive keys.

Reply:
Alright I updated it. I just renamed the file lol